client/src/constants.js

```diff
@@ -17,6 +17,10 @@ export const NAV_ITEMS = [
       { href: "/about", text: "Contact"} //Contact page not created yet, will send to About Us
     ],
   },
+  {
+    href: "/add-post",
+    text: "ADD POST",
+  },
```

Comment:
We can definitely put it in the Navbar for now for easy access but having a button for it in the future would be good 👍